app/page.tsx

```
@@ -1,8 +1,9 @@
-import ComingSoon from "./components/sections/ComingSoon";
+import LoginPage from "./LoginPage/page";
+import ComingSoon from "./ComingSoon/page";
```

ESLint

[error] 'ComingSoon' is defined but never used.

(@typescript-eslint/no-unused-vars)